Gruntfile.js

```diff
@@ -52,6 +52,7 @@ const bedrockDefaults = {
   customRoutes: 'modules/tinymce/src/core/test/json/routes.json',
   overallTimeout: 180000,
   singleTimeout: 60000,
+  polyfills: [ 'Promise', 'Symbol' ],
```

Comment:
So these are not default for all projects since some of the premium ones are still not using eslint and the promise rule?

Reply:
It's more some project might not be using ESLint (I was thinking the ReasonML code base for example), so we thought it was dangerous to turn the promise polyfill on by default. So after talking to the team we decided to make it explicit opt-in.

Reply:
I should add it could also apply to premium plugins, but all have been converted except clerk at this time.

Reply:
Nice. I will log a ticket to add eslint to clerk.